tests/conftest.py

```diff
@@ -14,7 +34,8 @@ def testing_images():
     async def _pull():
         docker = Docker()
         required_images = [
-            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine'
+            'alpine:latest', 'redis:latest', 'python:3.6.1-alpine',
+            'busybox:latest',
```

Comment:
Sure but then I will remove busybox from the `test_images.py` and use alpine instead. Fine by me but...

```
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
redis               latest              d4f259423416        13 days ago         106MB
busybox             latest              efe10ee6727f        2 weeks ago         1.13MB
python              3.6.1-alpine        ddd6300d05a3        5 weeks ago         88.7MB
alpine              latest              7328f6f8b418        5 weeks ago         3.96MB
```

In my humble opinion: the redis image is the most problematic. busybox is better than alpine but I don't really care and I believe we could/should also get rid of the Python image.